apps/emqx_conf/src/emqx_conf_schema.erl

```diff
@@ -1012,6 +1025,28 @@ fields("log_burst_limit") ->
                 }
             )}
     ];
+fields("log_throttling") ->
+    [
+        {time_window,
+            sc(
+                emqx_schema:duration_s(),
```

Comment:
Nit: since this is used by `send_after`, might be safer to use `timeout_duration_s`.

Reply:
Thanks @thalesmg, fixed: https://github.com/emqx/emqx/pull/12647